client/transactions/uncaptured/capture-authorization-button.tsx

```diff
@@ -0,0 +1,42 @@
+/** @format **/
```

Comment:
I think this button can be moved to another location for its reuse potential.

Reply:
Indeed, thanks. Done in [63d289f](https://github.com/Automattic/woocommerce-payments/pull/4676/commits/63d289fe6e9e4c89e21d197cf31ad4a1057524ef)